recipe-client-addon/lib/CleanupManager.jsm

```diff
@@ -4,6 +4,7 @@
 
 "use strict";
 
+const {utils: Cu} = Components;
```

Comment:
I don't know that it ever did. The changes that needed this are fairly new, and haven't been merged to mozilla-central yet, so it is entirely reasonable that they haven't really been tested.